manta-js/package/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "manta.js",
+  "version": "1.0.0",
+  "description": "manta.js sdk",
+  "main": "sdk.ts",
+  "dependencies": {
+    "@polkadot/api": "6.8.1",
```

Comment:
Might also want to configure a linter and add a lint script, you can maybe look at what the front end is doing